packages/manager/modules/metrics/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "@ovh-ux/manager-metrics",
+  "version": "0.0.0",
+  "private": true,
+  "description": "OVHcloud Metrics product",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/modules/metrics"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "main": "./src/index.js",
+  "dependencies": {
+    "lodash": "^4.17.15",
+    "moment": "^2.19"
+  },
+  "peerDependencies": {
+    "@ovh-ux/manager-config": "^0.4.0 || ^1.0.0",
```

It could be a dependency as it is not angular related. Moreover I think we need to
```suggestion
    "@ovh-ux/manager-config": " ^1.0.0 || ^2.0.0",
```